core/Mail/EmailStyles.php

```diff
@@ -89,6 +89,21 @@ class EmailStyles
      */
     public $brandNameLong;
 
+    /**
```

Comment:
are those changes supposed to be in here @diosmosis ?

Reply:
no, not sure how they got in, will remove